repos/system_upgrade/el7toel8/libraries/fetch.py

```diff
@@ -52,7 +81,7 @@ def read_or_fetch(filename, directory="/etc/leapp/files", service=None, allow_em
     cert = ("/etc/pki/consumer/cert.pem", "/etc/pki/consumer/key.pem")
     response = None
     try:
-        response = requests.get(service_path, cert=cert, proxies=proxies, timeout=REQUEST_TIMEOUT)
+        response = _request_data(service_path, cert=cert, proxies=proxies)
     except requests.RequestException as e:
         logger.error(e)
         _raise_error(local_path, "Could not fetch {f} from {sp} (unreachable address).".format(
```

Comment:
```
        _raise_error(local_path, "Could not fetch {f} from {sp} (unreachable address).".format(
```
'(unreachable address)' is probably misleading in case of ReadTimeout, no? I know it's unrelated, just happen to see this

Reply:
It is. I know about that. But anyway, it's already planned to do bunch of changes (refactoring, ....) around, so I vote to keep the message for now and fix it in the follow up (not sure whether the change of the message will not break the tests again..)